apps/dialtone-documentation/docs/about/whats-new/posts/2024-12-10.md

```diff
@@ -0,0 +1,43 @@
+---
+heading: Recipe class renames (POSSIBLE BREAKING CHANGE)
+author: Brad Paugh
+posted: '2024-12-10'
+---
+
+<BlogPost :author="$frontmatter.author" :posted="parse($frontmatter.posted, 'y-M-d', new Date())" :heading="$frontmatter.heading">
+
+Hi again, similar to our previous component class renames, we've now renamed some classes in our recipes to be consistent with the naming of the rest of our components. Similar to last time this will only be a breaking change if you are manually referencing these internal classes in your application. The fix is the same as last time, just find / replace the old class name to the new class name in your consuming application.
+
+Note that this will also include all sub elements of these classes, so if it says to rename `dt-list-item` to `d-list-item` this will also be the case for `dt-list-item--focusable`
```

Comment:
change this one for an example with a recipe please

Reply:
Updated.